helm/dashboards/charts/public_dashboards/dashboards/shared/public/cluster-overview.json

```diff
@@ -1566,7 +1566,7 @@
   "timepicker": {},
   "timezone": "browser",
   "title": "Cluster Overview",
-  "uid": "deanir14wf6rkf",
+  "uid": "gs_cluster-overview",
```

Comment:
Why the underscore? Why not simply a dash?

It's not a big deal, but experience shows that the more complicated URLs get, the easier some mistakes will sneak in.